### PR TITLE
Spanner plugin e2e test scenarios

### PR DESCRIPTION
**Pull request by sawantpritam**
@tivv @rmstar @itsankit-google Please review.

### CODE REVIEW

File: src/e2e-test/java/io/cdap/plugin/spanner/actions/CdfSpannerActions.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+    public static void enterImportQuery(String query) {
+        CdfSpannerLocators.importQuery.sendKeys(query);
+    }
+
+    public static void clickValidateButton() {
+        CdfSpannerLocators.validateButton.click();
+    }
+
+    public static void enterTablename(String tableName) {
```
Comment (itsankit-google) [inline src/e2e-test/java/io/cdap/plugin/spanner/actions/CdfSpannerActions.java]: enterTablename->enterTableName

File: src/e2e-test/java/io/cdap/plugin/spanner/actions/CdfSpannerActions.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        CdfSpannerLocators.spannerTableName.sendKeys(tableName);
+    }
+
+    public static void enterDatabaseename(String databaseName) {
```
Comment (itsankit-google) [inline src/e2e-test/java/io/cdap/plugin/spanner/actions/CdfSpannerActions.java]: enterDatabasename->enterDatabaseName

File: src/e2e-test/java/io/cdap/plugin/spanner/actions/CdfSpannerActions.java (hunk 1 above)
Comment (sawantpritam) [reply src/e2e-test/java/io/cdap/plugin/spanner/actions/CdfSpannerActions.java]: Changed.

File: src/e2e-test/java/io/cdap/plugin/spanner/actions/CdfSpannerActions.java (hunk 2 above)
Comment (sawantpritam) [reply src/e2e-test/java/io/cdap/plugin/spanner/actions/CdfSpannerActions.java]: Changed.

**Comment by sawantpritam** [comment]
Spanner e2e test cases added in https://github.com/data-integrations/google-cloud/pull/893

**Pull request closed without merging**